src/modules/navigation/components/MaterialRequestCenterBlade/MaterialRequestCenterBlade.tsx

```diff
@@ -31,11 +37,29 @@ const MaterialRequestCenterBlade: FC<MaterialRequestCenterBladeProps> = ({ isOpe
 	const { tHtml, tText } = useTranslation();
 	const dispatch = useDispatch();
 
-	const [isPersonalInfoBladeOpen, setIsPersonalInfoBladeOpen] = useState(false);
-	const [isEditMaterialRequestBladeOpen, setIsEditMaterialRequestBladeOpen] = useState(false);
 	const [selectedMaterialRequest, setSelectedMaterialRequest] = useState<MaterialRequest | null>(
 		null
 	);
+	const [activeBlade, setActiveBlade] = useState<MaterialRequestBladeId>(
+		MaterialRequestBladeId.Overview
+	);
+	const getCurrentLayer = () => {
+		console.log('get current layer: ', { isOpen, activeBlade });
+		if (!isOpen) {
+			return 0;
+		}
+		switch (activeBlade) {
+			case MaterialRequestBladeId.Overview:
+				return 1;
+
+			case MaterialRequestBladeId.EditMaterialRequest:
```

Comment:
Missing case execution?

Reply:
no this is a fall through case
the same thing has to happen for both cases